tasks/closedSourceRationaleCheck.ts

```diff
@@ -0,0 +1,39 @@
+import { danger } from "danger"
+
+export const target = "Rationale for Closed Source"
+export const message = "missing rationale for closed source"
+
+const parseReadme = async (getter, args) => {
+  const { data: contents } = await getter(args)
+  const readme = Buffer.from(contents.content, "base64").toString("utf8")
+  return readme
+}
+
+const findMissing = (repos, getter, org) => {
+  const missingRationale = repos.filter(async repo => {
+    try {
+      const args = { owner: org, repo: repo.name, path: "README" }
+      const readme = await parseReadme(getter, args)
+      return !readme.includes(target)
+    } catch (error) {
+      return false
+    }
+  })
+
+  return Promise.all(missingRationale)
+}
+
+const openIssue = async (creater, args) => {
+  await creater("rationale-check", message, args)
+}
+
+export default async () => {
+  const api = danger.github.api
+  const org = "artsy"
+  const { data: repos } = await api.repos.getForOrg({ org, type: "private", per_page: 100 })
+
+  const missingRationale = await findMissing(repos, api.repos.getContent, org)
+
+  const args = { open: true, owner: org, repo: "0", title: message }
+  missingRationale.forEach(repo => openIssue(danger.github.utils.createUpdatedIssueWithID, args))
```

Comment:
This will create ~100 issues in parallel, which might end up overwhelming the rate limit

Reply:
What do you think we should do instead?